doc/2/api/controllers/security/search-users-by-credentials/index.md

```diff
@@ -0,0 +1,114 @@
+---
+code: true
+type: page
+title: searchUsersByCredentials
+---
+
+# searchUsersByCredentials
+
+<SinceBadge version="auto-version"/>
+
+Given a credentials related search query, returns matched users' kuid.
+Since credentials depend on the authentification strategy, so do the search query.
+
+::: info
+If you are using a custom strategy plugin, you must first implement the optional search method in order to use this action.
+:::
+
+::: warning
+This method is not intended to be exposed to the end user because of sensitive data that represents an exhaustive list of users ids.
+:::
+
+---
+
+## Query Syntax
+
+### HTTP
+
+```http
+URL: http://kuzzle:7512/credentials/<strategy>/users/_search
+Method: POST
+Body:
+```
+
+```js
+{
+  "bool": {
+    "must": [
+      {
+        "match": {
+          // example with the "local" authentication strategy
+          "username":  "test@example.com"
+        }
+      }
+    ]
+  }
+}
+```
+
+### Other protocols
+
+```js
+{
+  "controller": "security",
+  "action": "searchUsersByCredentials",
+  "strategy": "<strategy>",
+  "body": {
```

Comment:
IMHO the body should be what we call a `searchBody`, to allows users to pass either `query` and/or `sort` properties